pxtblocks/fields/field_tilemap.ts

```diff
@@ -92,24 +95,59 @@ namespace pxtblockly {
                     const old = this.getValue();
 
                     this.state = result;
+                    this.state.projectReferences = null;
+
+                    const project = pxt.react.getTilemapProject();
+
+                    const lastRevision = project.revision();
+                    project.pushUndo();
+
+                    if (result.deletedTiles) {
+                        for (const deleted of result.deletedTiles) {
```

Comment:
Minor regression (from beta -> this pr's build) I noticed while testing here; after duplicating a tilemap with a custom tile it's impossible to delete the tile from the tilemap editor. You get stuck on the `Unable to delete` message when trying to delete the tile, even after erasing that tile from both tilemaps

![image](https://user-images.githubusercontent.com/5615930/86955335-433c1200-c10c-11ea-8d95-85c57dfb7327.png)

Reply:
Fixed